src/lib/edge-functions/registry.mjs

```diff
@@ -256,7 +289,9 @@ export class EdgeFunctionsRegistry {
       return
     }
 
-    log(`${NETLIFYDEVLOG} ${chalk.magenta('Reloading')} edge functions...`)
+    const reason = this.#debug ? ` because ${chalk.underline(path)} has changed` : ''
```

Comment:
This is particularly useful when debugging failures in the CI, but I can also see it being useful when troubleshooting strange behaviour with file watchers.